yarn-project/stdlib/src/p2p/consensus_payload.ts

```diff
@@ -41,10 +41,20 @@ export class ConsensusPayload implements Signable {
   }
 
   getPayloadToSign(domainSeparator: SignatureDomainSeparator): Buffer {
-    const abi = parseAbiParameters('uint8, (bytes32, bytes, (uint256), bytes32, bytes32[])');
+    const abi = parseAbiParameters(
+      'uint8, ' + //domainSeperator
+        '(' +
+        'bytes32, ' + // archive
+        '((bytes32,uint32),((bytes32,uint32),(bytes32,uint32),(bytes32,uint32))), ' + // stateReference
```

Comment:
Thats one ugly looking line 😬 Don't we have any neat way we could get this thing directly from the abis or so instead 🤔 Or guess it would just explode so maybe not too big a deal if easy to spot.